tests/rustdoc-ui/issues/issue-107918.rs

```diff
@@ -1,6 +1,7 @@
 // aux-build:panic-handler.rs
 // compile-flags: --document-private-items
 // build-pass
+// ignore-macos
```

Comment:
Wouldn't it be simpler to just run it on linux instead?

Reply:
Simpler as in fewer lines, yes.

However, I don't have the context of the original point of the test so I can't speak to if the test is *supposed* to only be run on Linux. `ONLY Linux` and `NOT (macOS OR Windows)` produce different sets of platforms.

Reply:
> Simpler as in fewer lines, yes.

More like "we don't want to check that each platform doesn't break".

> However, I don't have the context of the original point of the test so I can't speak to if the test is _supposed_ to only be run on Linux. `ONLY Linux` and `NOT (macOS OR Windows)` produce different sets of platforms.

The point of this test is to be a regression test for the #107918 issue. So as long as it's working on one platform, we're achieving this goal. However to do so, we use very uncommon code, so I think it's better to reduce the number of targets on which we actually check it for simpler maintenance.

So yes, please change it into only linux. :)

Reply:
Changed!

Reply:
Thanks!